book/section4/mediator.md

```diff
@@ -2,22 +2,132 @@
 
 ## Set up a Mediator
 
-We will teach the reader how to
+As usual, we use Docker to setup our infrastructure, `mediator` is no different and here is an example `docker-compose.yml`, `mongo-initdb.js` and `.env` files, you should put all 3 files in a folder.
 
-- Install, configure, and Run your own Mediator
-- How to manage web sockets
-- Performance Tuning
+`docker-compose.yml`
+```yml
+services:
+  ################
+  ### MEDIATOR ###
+  ################
+
+  mediator-mongo:
+    image: mongo:6.0
+    command: [ "--auth" ]
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=admin
+      - MONGO_INITDB_ROOT_PASSWORD=admin
+      - MONGO_INITDB_DATABASE=mediator
+    volumes:
+      - ./mongo-initdb.js:/docker-entrypoint-initdb.d/initdb.js
+
+  mediator:
+    image: docker.io/identus/identus-mediator:${MEDIATOR_VERSION:latest}
+    ports:
+      - "${MEDIATOR_PORT:-8080}:8080"
+    environment:
+      # Creates the identity:
+      # These keys are for demo purpose only for production deployments generate keys
+      # Please follow the README file in the Mediator repository for guidelines on How to generate JWK format keys
+      # KEY_AGREEMENT KEY_AUTHENTICATION are using format JOSE(JWK) OKP type base64urlsafe encoded keys
+      - KEY_AGREEMENT_D=${KEY_AGREEMENT_D:-Z6D8LduZgZ6LnrOHPrMTS6uU2u5Btsrk1SGs4fn8M7c}
+      - KEY_AGREEMENT_X=${KEY_AGREEMENT_X:-Sr4SkIskjN_VdKTn0zkjYbhGTWArdUNE4j_DmUpnQGw}
+      - KEY_AUTHENTICATION_D=${KEY_AUTHENTICATION_D:-INXCnxFEl0atLIIQYruHzGd5sUivMRyQOzu87qVerug}
+      - KEY_AUTHENTICATION_X=${KEY_AUTHENTICATION_X:-MBjnXZxkMcoQVVL21hahWAw43RuAG-i64ipbeKKqwoA}
+      - SERVICE_ENDPOINTS=${SERVICE_ENDPOINTS:-http://identus-mediator:8080;ws://identus-mediator:8080/ws}
+      - MONGODB_USER=admin
+      - MONGODB_PASSWORD=admin
+      - MONGODB_PROTOCOL=mongodb
+      - MONGODB_HOST=mediator-mongo
+      - MONGODB_PORT=27017
+      - MONGODB_DB_NAME=mediator
+    depends_on:
+      - "mediator-mongo"
+```
+
+`mongo-initdb.js`
+```js
+db.createUser({
+    user: "admin",
+    pwd: "admin",
```

Comment:
Are we going to call out somewhere that this is insecure?  Maybe in the Production setup chapter?